ui/raidboss/raidemulator.js

```diff
@@ -231,67 +231,124 @@ class LogCollector {
+        12960, 12963, 12966, 12977, 12979, 12990, 12995, 12998, 12999, 14476, 14478, 16473, 16474,
+        16475, 16476, 17674, 17675, 17676, 17677, 17719, 17720, 17721, 17722, 17723, 17724, 17725,
+        17726, 3543, 3545, 3546, 3547, 4262, 4287, 4288, 50160, 50161, 50245, 50273, 50274, 63, 70,
+        71, 7394, 7395, 7396, 74, 8780, 8781, 8782, 8783, 8784, 8785, 8787, 8789, 8925,
       ],
       DRG: [
-        75, 78, 79, 81, 83, 84, 85, 86, 87, 88, 90, 92, 94, 95, 96,
-        3553, 3554, 3555, 3556, 3557, 7397, 7398, 7399, 7400,
+        16477, 16478, 16479, 16480, 17728, 17729, 3553, 3554, 3555, 3556, 3557, 4292, 4293, 50162,
+        50163, 50247, 50275, 50276, 7397, 7398, 7399, 7400, 86, 8791, 8792, 8793, 8794, 8795,
+        8796, 8797, 8798, 8799, 8802, 8803, 8804, 8805, 8806, 92, 94, 95, 96, 9640,
       ],
       NIN: [
-        2240, 2241, 2242, 2245, 2246, 2247, 2248, 2253, 2254, 2255, 2256, 2257, 2258, 2259,
-        2260, 2261, 2262, 2263, 2264, 2265, 2266, 2267, 2268, 2269, 2270, 2271, 2272, 3563,
-        3564, 3565, 3566, 3567, 7401, 7402, 7403,
+        16488, 16489, 16491, 16492, 16493, 17413, 17414, 17415, 17416, 17417, 17418, 17419, 17420,
+        17732, 17733, 17734, 17735, 17736, 17737, 17738, 17739, 2246, 2259, 2260, 2261, 2262,
+        2263, 2264, 2265, 2266, 2267, 2268, 2269, 2270, 2271, 2272, 3563, 3566, 4295, 50165,
+        50166, 50167, 50250, 50279, 50280, 7401, 7402, 7403, 8807, 8808, 8809, 8810, 8812, 8814,
+        8815, 8816, 8820, 9461,
       ],
       SAM: [
-        7477, 7478, 7479, 7480, 7481, 7482, 7483, 7484, 7485, 7486, 7487, 7488, 7489, 7490,
-        7491, 7492, 7493, 7494, 7495, 7496, 7497, 7498, 7499, 7500, 7501, 7502, 7867,
+        16481, 16482, 16483, 16484, 16485, 16486, 16487, 17740, 17741, 17742, 17743, 17744, 50208,
+        50215, 50277, 50278, 7477, 7478, 7479, 7480, 7481, 7482, 7483, 7484, 7485, 7486, 7487,
+        7488, 7489, 7490, 7491, 7492, 7493, 7494, 7495, 7496, 7497, 7498, 7499, 7501, 7502, 7855,
+        7857, 7867, 8821, 8822, 8823, 8824, 8825, 8826, 8828, 8829, 8830, 8831, 8833,
       ],
       BRD: [
-        97, 98, 100, 101, 103, 106, 107, 110, 112, 113, 114, 115, 116, 117, 118, 3558, 3559,
-        3560, 3561, 3562, 7404, 7405, 7406, 7407, 7408, 7409,
+        10023, 114, 116, 117, 118, 13007, 14479, 16494, 16495, 16496, 17678, 17679, 17680, 17681,
+        17682, 17745, 17747, 3558, 3559, 3560, 3561, 3562, 4294, 50168, 50169, 50282, 50283, 50284,
+        50285, 50286, 50287, 7404, 7405, 7406, 7407, 7408, 7409, 8836, 8837, 8838, 8839, 8841,
+        8842, 8843, 8844, 9625,
       ],
       MCH: [
-        2864, 2865, 2866, 2867, 2868, 2870, 2872, 2873, 2874, 2875, 2876, 2878, 2879, 2880,
-        2881, 2885, 2887, 2888, 2890, 3487, 7410, 7411, 7412, 7413, 7414, 7415, 7418, 9015, 9372,
+        16497, 16498, 16499, 16500, 16501, 16502, 16503, 16504, 16766, 16889, 17206, 17209, 17749,
```

Comment:
16504 is 0x4078, arm punch from the Automata Queen.

Unfortunately, both this and the sch codes (and probably drk and nin) include ability codes from the seraph, selene, and automaton queen.  This leads to the discovery of a ton of players called Automaton Queen and Selene which are not assigned to the original player who summoned them.